portfolio/pom.xml

```diff
@@ -1,4 +1,4 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
+ <project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Unnecessary change

Reply:
I think there was an extra blank space or line break at the beginning of the file, so I think in this change I just deleted that.

Reply:
Can we remove the unnecessary space in the beginning of this line?